state/futures/reducer.ts

```diff
@@ -316,6 +319,30 @@ const futuresSlice = createSlice({
 			};
 		});
 
+		// margin transfers
+		builder.addCase(fetchMarginTransfers.pending, (futuresState) => {
+			futuresState.queryStatuses.dailyVolumes = LOADING_STATUS;
+		});
+		builder.addCase(fetchMarginTransfers.fulfilled, (futuresState, { payload }) => {
+			futuresState.queryStatuses.isolatedMarginTransfers = SUCCESS_STATUS;
+			if (payload) {
+				const { wallet, network, accountType } = payload;
+				if (accountType === 'cross_margin') {
+					futuresState.crossMargin.accounts[network][wallet].marginTransfers =
```

Comment:
Could do a single line here rather than if else with:
```
futuresState[accountType(payload.accountType)].accounts[network][wallet].marginTransfers = payload.marginTransfers
```